src/algebra/order/to_interval_mod.lean

```diff
@@ -341,115 +346,108 @@ section Ico_Ioc
 variables (a b)
 
 omit hα
-/-- `mem_Ioo_mod p a b` means that `b` lies in the open interval `(a, a + p)` modulo `p`.
-Equivalently (as shown below), `b` is not congruent to `a` modulo `p`, or `to_Ico_mod hp a` agrees
-with `to_Ioc_mod hp a` at `b`, or `to_Ico_div hp a` agrees with `to_Ioc_div hp a` at `b`. -/
-def mem_Ioo_mod (p a b : α) : Prop := ∃ z : ℤ, b - z • p ∈ set.Ioo a (a + p)
+/-- `Imodeq p a b` means that `b` does not lie in the open interval `(a, a + p)` modulo `p`.
+Equivalently (as shown below), `b` is congruent to `a` modulo `p`, or `to_Ico_mod hp a` disagrees
+with `to_Ioc_mod hp a` at `b`, or `to_Ico_div hp a` disagrees with `to_Ioc_div hp a` at `b`. -/
+def Imodeq (p a b : α) : Prop := ∀ z : ℤ, b - z • p ∉ set.Ioo a (a + p)
```

Comment:
Calling this definition `Imodeq` sounds weird to me. If you want to unify with `smodeq` in the future, wouldn't you take the RHS of `Imodeq_iff_eq_mod_zmultiples` as the definition?

Reply:
> If you want to unify with `smodeq` in the future, wouldn't you take the RHS of `Imodeq_iff_eq_mod_zmultiples` as the definition?

Yes, absolutely; I just figured that was a much larger refactor than I wanted to make, as it flips the order of all the proofs below. If you want to do this in a follow-up, I'm happy to review; but we could also just leave it till after the port.

Reply:
`Another possible name would be `archimedian.modeq`?

Reply:
Or `add_comm_group.modeq`

Reply:
FYI I am currently writing the API for
```
def modeq (p a b : α) : Prop := ∃ z : ℤ, b - a = z • p
```